components/movementsensor/dualgps/dualgps.go

```diff
@@ -114,14 +115,31 @@ func (dg *dualGPS) Reconfigure(ctx context.Context, deps resource.Dependencies,
 	if err != nil {
 		return err
 	}
+
+	firstProps, err := first.Properties(ctx, nil)
+	if err != nil {
+		return err
+	}
 	dg.gps1 = first
 
 	second, err := movementsensor.FromDependencies(deps, newConf.Gps2)
 	if err != nil {
 		return err
 	}
+	secondProps, err := first.Properties(ctx, nil)
+	if err != nil {
+		return err
+	}
 	dg.gps2 = second
 
+	if !firstProps.PositionSupported || !secondProps.PositionSupported {
+		return fmt.Errorf(
+			"configured movement sensors %v and %v do not support reporting their Position, they cannot be used for dual-gps calculations",
+			first.Name().ShortName(),
+			second.Name().ShortName(),
+		)
+	}
```

Comment:
```suggestion
        var errs error
	if !firstProps.PositionSupported {
		multierr.combine(errs, fmt.Errorf("configured movement sensor %v does not support reporting its Position, it cannot be used for dual-gps calculations", first.Name().ShortName())
	}
	if !secondProps.PositionSupported {
		multierr.combine(errs, fmt.Errorf("configured movement sensor %v does not support reporting its Position, it cannot be used for dual-gps calculations", second.Name().ShortName())
	}
	if errs != nil {
	        return errs
	}
```

Reply:
or if you'd like, add a third option for if both don't support position if you don't want there to be two errors in that case. I think this makes debugging easier. but if it's too much code for your liking, the original is fine